.env

```diff
@@ -7,10 +7,11 @@ TEKU_DOCKER_TAG=21.1.1
 NIMBUS_DOCKER_TAG=v1.0.6
 VOUCH_DOCKER_TAG=1.0.3
 DIRK_DOCKER_TAG=1.0.1
-LODESTAR_DOCKER_TAG=v0.12.0
+LODESTAR_DOCKER_TAG=v0.14.0-3
 
 GRAFANA_DOCKER_TAG=7.3.7
 GRAFANA_IMAGE_RENDERER_TAG=2.0.0
 PROMETHEUS_DOCKER_TAG=v2.22.2
+PROMETHEUS_NODE_EXPORTER_DOCKER_TAG=master
```

Comment:
can we have a specific version here? `master` is usually no stable release

Reply:
yeah, I've changed it to v1.0.1